tools/topology/topology2/cavs/CMakeLists.txt

```diff
@@ -44,6 +44,11 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-cavs-tgl-nocodec.bin"
 # SSP topology for MTL
 "cavs-nocodec\;sof-mtl-nocodec\;PLATFORM=mtl,NUM_DMICS=2,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-ace-mtl-nocodec.bin"
+
+# SSP topology for MTL
+"cavs-rt5682\;sof-mtl-max98357a-rt5682\;PLATFORM=mtl,\
+NUM_HDMIS=4,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-ace-mtl-nocodec.bin"
```

Comment:
Zero dmics on this platform?

Reply:
There are two reasons I did not put DMIC
1. The build will fail as reported in https://github.com/thesofproject/sof/issues/6007
2. On our test RVP with I2S config, DMIC option is not available, so it does not hurt.

Maybe I can change sof-mtl-max98357a-rt5682 to sof-mtl-max98357a-rt5682-no-dmic?

Reply:
@yongzhi1 
a) we should fix known errors first, not work-around them
b) if this is a test topology, it needs to go in a 'development' folder, so that it's clear we don't intend to release it to the general public. A device without microphone isn't very useful, is it? Well, unless it's a feature for security/privacy usages....

Reply:
Got it, let me wait for one week and hopefully we can figure out the build issue, if not, will send it to development folder.

Reply:
> figure out the build issue, if not, will send it to development folder.

Do you mean temporarily, as staging area until it's fixed? Most other topologies in the development folder will _never_ be released in https://github.com/thesofproject/sof-bin because they're not meant to be used in production on any device.

The difference between the development folder and the other, released folder is usually not a "quality" difference. It's usually a difference between something used in production on someone's device versus not.

Reply:
> > figure out the build issue, if not, will send it to development folder.
> 
> Do you mean temporarily, as staging area until it's fixed? Most other topologies in the development folder will _never_ be released in https://github.com/thesofproject/sof-bin because they're not meant to be used in production on any device.
> 
> The difference between the development folder and the other, released folders is usually not a "quality" difference. It's usually a difference between something used in production on customer's device versus not.

Then the development folder should be the right place since the tplg is intended for RVP only not product, think the nocodec tplg etc. should also be moved to the same folder.